### ISSUE TITLE
Uploading a docx file to the knowledge will result in an error.

### ISSUE THREAD

**Issue by kkjinping** [langgenius/dify#9592]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.10.0

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

Uploading a docx file to the knowledge will result in an error：

```
__init__() missing 1 required keyword-only argument: 'created_by_role'
```

### ✔️ Expected Behavior

no error

### ❌ Actual Behavior

error

**Comment by crazywoola** [comment]
Already fixed in #9575

**Comment by crazywoola** [comment]
https://github.com/langgenius/dify/pull/9607
